scripts/md2rst.py

```diff
@@ -29,5 +29,5 @@ def download_pandoc():
 def md2rst(text):
     download_pandoc()
     output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('filters', 'yml_filter.py')])
+     filters=[os.path.join(os.getcwd(),'filters', 'yml_filter.py')])
```

Comment:
@imujjwal96 could you test this please. I am not able to set environment variables on my test repository because of https://github.com/travis-ci/travis-ci/issues/7831 so can't test this now.